content/en/deployment_gates/setup.md

```diff
@@ -92,12 +93,18 @@ This rule type uses Watchdog's [APM Faulty Deployment Detection][1] to compare t
 
 ## Evaluate a Deployment Gate
 
-After a gate is configured with at least one rule, you can evaluate the gate while deploying the related service with an API call: 
+Deployment Gate evaluations are asynchronous, as the evaluation process can take some time to complete. This means that when you trigger an evaluation, the a process is started in the background and provides you with an evaluation ID to track its progress.
+
+- First, request a deployment gate evaluation, which initiates the process and returns an evaluation ID.
+- Then, poll the evaluation status endpoint using the evaluation ID to retrieve the result once the evaluation is complete.
```

Comment:
```suggestion
- Then, periodically poll the evaluation status endpoint using the evaluation ID to retrieve the result once the evaluation is complete.
```

Reply:
In general, I feel that we could expand a bit more over the approach that we require from customers. It needs to be clear that the second endpoint needs to be called periodically, as the duration of the gate might vary

Reply:
Makes sense! At the moment we check monitors every 15 seconds (this could change in the future), so polling the endpoint every 10-15 seconds or so would be optimal to reduce delays in getting an evaluation result. I'll update the doc!